src/libcore/cmp.rs

```diff
@@ -625,6 +627,7 @@ pub trait PartialOrd<Rhs: ?Sized = Self>: PartialEq<Rhs> {
     /// let result = std::f64::NAN.partial_cmp(&1.0);
     /// assert_eq!(result, None);
     /// ```
+    #[must_use]
```

Comment:
This method is the only newly-annotated one that's not an operator.  Intentional?

(I'm by no means against it, and see no situation where it being unused would be good, but I recall the RFC intentionally leaving it off most places for now.)